### ISSUE TITLE
Iron chests disappear from inventory and when placed.

### ISSUE THREAD

**Issue by Doubletap** [progwml6/ironchest#199]
## Version Information
`Minecraft: 1.14.4'
`Forge: 28.0.104'
Iron Chests:9.0.4'

## Report Information
I can see the chests in creative mode, but once I place one it disappears. They will also disappear from my inventory. The mod is installed on a server and works fine when loaded into a client.

**Comment by alexbegt** [comment]
I am unable to recreate this, creating a custom server and placing the chest in the world leaves it where i placed it and does not remove it from my inventory; What other mods are you running?

**Comment by Doubletap** [comment]
Xearo’s mini map mod I have upgraded to 9.1.5 and forge 28.1.0. it allows me to place the shulker boxes but none of the iron through obsidian.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10


________________________________

**Comment by Enigma416** [comment]
I am having this same issue.

Just created a home server (1.14.4) for me and my wife. Ported over the mods and save files for a mid-progress game and everything is working as it should, however all my mod-chests (different tiers of the iron chest mod) have vanished entirely with the stuff inside.

i couldnt find any error in my startup log, and i can find and select these chests in creative, but when i try to place them it disapears nearly immediately. i've even tried moving over 1k blocks in any directions to test for server protections and nothing.

do you know what could be causing this? I am using 7 mods in total that work and have been shown to be compatible for dozens of hours of LAN Survival gameplay.


Mod list:
Forge
Heres what your looking at (HWYLA)
Just another rotten flesh to leather mod (JRFTL)
Ruins
Just Enough Items (JEI)
Xaero's Minimap
Xaero's World map
Iron Chest

**Comment by alexbegt** [comment]
Can you upload the debug.log from the server?

**Comment by Enigma416** [comment]
I apologize, I'm new to making a server. I don't know how to make/get debug.log
I tried already. Here is my Server log from startup, hopefully that helps for now while i try and figure out how to get my debug.log
[ServerLog-Enigma416.txt](https://github.com/progwml6/ironchest/files/3872576/ServerLog-Enigma416.txt)

**Comment by alexbegt** [comment]
Are you running the server from your actual world save? (like the local server) or running it using the minecraft_server.jar?

**Comment by Enigma416** [comment]
It was running fine from world save (local server), i then opted to create a dedicated server JAR with the same save file. world *seemingly* started fine but as i stated prior, the Iron chest mod chests disappear including their contents.

TLDR: the problem is occurring on my minecraft_server.jar

**Comment by Enigma416** [comment]
i stried doing a debug start/stop with commands. hope this is the right thing.
[profile-results-2019-11-20_21.51.21.txt](https://github.com/progwml6/ironchest/files/3872741/profile-results-2019-11-20_21.51.21.txt)



also to note, i am using forge 28.1.44 /minecraft version 1.14.4

**Comment by alexbegt** [comment]
Did you happen to start this world with a mod that you ended up removing? Also, Could you update to the latest forge?

**Comment by Enigma416** [comment]
1st question, no i did not add/remove any mods since the creation of the game.
2nd question, i can try that next later today and see if that works
